src/content/reference/react/captureOwnerStack.md

```diff
@@ -0,0 +1,164 @@
+---
+title: captureOwnerStack
+---
+
+<Wip>
+
+**This API is experimental and is not available in a stable version of React yet.**
+
+You can try it by upgrading React packages to the most recent experimental version:
+
+- `react@experimental`
+
+Experimental versions of React may contain bugs. Don't use them in production.
+
+</Wip>
+
+<Intro>
+
+`captureOwnerStack` reads the current **owner** Component stack and returns it as a string if available.
```

Comment:
We should make it more clear that this is dev-only. I would mention it here, in a Caveat, and a Troubleshooting "My owner stacks are empty in production" section.

Reply:
Did all that. Also explained this in the createRoot and hydrateRoot usage examples. Though possible I missed some spots.